themes/icons/check.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+<path d="M15.175 4L6.175 13L1.185 8.05" stroke="#545B65" stroke-width="2" stroke-miterlimit="10"/>
```

Comment:
The icon svg needs refinement

1. Remove hard-coded color because it is not dark mode compatible
2. Replace strokes with shapes (you need to ask designers to create another export)